include/engine/guidance/assemble_steps.hpp

```diff
@@ -109,6 +116,7 @@ inline std::vector<RouteStep> assembleSteps(const datafacade::BaseDataFacade &fa
                                           NO_ROTARY_NAME,
                                           segment_duration / 10.0,
                                           distance,
```

Comment:
is there a reason why we store distance in meters and duration/weight in 10ths of (seconds/number)? Shouldn't we keep this consistent?